RecoTracker/TkTrackingRegions/plugins/PointSeededTrackingRegionsProducer.h

```diff
@@ -196,9 +247,10 @@ class  PointSeededTrackingRegionsProducer : public TrackingRegionProducer
   int m_maxNRegions;
   edm::EDGetTokenT<reco::VertexCollection> token_vertex; 
   edm::EDGetTokenT<reco::BeamSpot> token_beamSpot; 
-  double eta_input, phi_input;
   int m_maxNVertices;
 
+  std::vector<edm::ParameterSet> points;
```

Comment:
hi @JanFSchulte - please store a vector of eta and phi values instead of the ParameterSets. This will avoid many string comparisons (and save space)